package.json

```diff
@@ -4,23 +4,23 @@
   "description": "Utility library that extracts the CLDR data that's needed by the FormatJS libraries.",
   "main": "index.js",
   "scripts": {
-    "test": "istanbul cover -- _mocha test/unit/ --reporter spec"
+    "test": "nyc mocha test/unit/ --reporter spec"
   },
   "author": "Eric Ferraiuolo <edf@ericf.me>",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "cldr-core": "^34.0.0",
-    "cldr-dates-full": "^34.0.0",
-    "cldr-numbers-full": "^34.0.0",
-    "glob": "^5.0.1",
-    "make-plural": "^2.1.3",
+    "cldr-core": "35",
```

Comment:
Why use 35 versus ^?

Reply:
it's the same really. I'll prob have to revert to 34 until the upstream CLDR issue for `kw` locale is resolved
https://github.com/unicode-org/cldr/pull/6